manifest/oci.go

```diff
@@ -220,7 +220,12 @@ func (m *OCI1) Inspect(configGetter func(types.BlobInfo) ([]byte, error)) (*type
 		Architecture:  v1.Architecture,
 		Os:            v1.OS,
 		Layers:        layerInfosToStrings(m.LayerInfos()),
+		LayersDetail:  imgInspectLayersFromLayerInfos(m.LayerInfos()),
```

Comment:
Non-blocking: It would be nice to call `LayerInfos` only once.

(In all three format handlers.)